build.gradle

```diff
@@ -343,17 +343,29 @@ task cmakeJniLib(type:Exec) {
         args.add("-DBLAS_LIBRARIES=$rootDir\\src\\main\\resources\\windowsDependencies\\libopenblas.dll")
         args.add("-DLAPACK_LIBRARIES=$rootDir\\src\\main\\resources\\windowsDependencies\\libopenblas.dll")
     }
+    //if (System.getenv("LIBNAME") == "nmslib_linux_x64") {
+    //    environment('CC', 'gcc10-gcc')
+    //    environment('CXX', 'gcc10-g++')
+    //    environment('FC', 'gcc10-gfortran')
+    //}
 
     commandLine args
 }
 
+task buildNmslib(type:Exec) {
+    dependsOn cmakeJniLib
+    workingDir 'jni'
+    commandLine 'make', '-B', 'opensearchknn_nmslib', '-j', "${nproc_count}"
```

Comment:
what does `-B` do ?

Reply:
Will remove, but basically during test this is to force a rebuild even if it is already built.